django_test_migrations/migrator.py

```diff
@@ -65,12 +66,18 @@ def apply_tested_migration(self, targets: MigrationSpec) -> ProjectState:
         return self._migrate(normalize(targets))
 
     def reset(self) -> None:
-        """Reset the state to the most recent one."""
+        """
+        Reset the state to the most recent one.
+
+        Notably, signals are not muted here to avoid
+        https://github.com/wemake-services/django-test-migrations/issues/128
+        """
```

Comment:
```suggestion
        """ Reset the state to the most recent one.

        Notably, signals are not muted here to avoid
        https://github.com/wemake-services/django-test-migrations/issues/128
        
        """
```

Reply:
We actually use 

```
"""
Reset the state to the most recent one.
```

style 🙂

Reply:
sorry, my bad - I've corrected the suggestion's code